### PR TITLE
Added address field to event type

### PR DESCRIPTION
**Pull request by AndrewBestbier**

### CODE REVIEW

**Comment by lpil** [comment]
Is address descriptive enough?

Also needs a location link which will be used for a google maps link on react.london events

**Comment by AndrewBestbier** [comment]
@lpil Do you need co-ords?  I know in prismic we store both the event address and event co-ords. I should probably make it query able for the Api anyway.

**Comment by jameshopkins** [comment]
Coordinates will be super useful when implement things like OpenGraph and Schema Org

**Comment by lpil** [comment]
Yes please, we need them on React London for the google maps link

Can built it like so: http://www.google.com/maps/place/49.46800006494457,17.11514008755796

File: lib/event/eventFields.js [hunk 1]
```diff
@@ -76,6 +77,10 @@ export const EventType = new GraphQLObjectType({
       type: GraphQLString,
       description: 'Filename of an image uploaded to a 3rd party hosting service',
     },
+    location: {
+      type: LocationType,
+      description: 'The address of the event in both postal and coordinate form',
```
Comment (lpil) [inline lib/event/eventFields.js]: Is this correct?

**Comment by lpil** [comment]
Looks good :)

File: lib/event/eventFields.js (hunk 1 above)
Comment (AndrewBestbier) [inline lib/event/eventFields.js]: Yup. I have broken it further down now so it is now:
location -> address, coordinates -> lat, long

**Pull request merged**